src/components/PostCompletionViews/ConfirmationView.stories.js

```diff
@@ -1,15 +1,15 @@
 import {expect} from '@storybook/jest';
-import {within} from '@storybook/testing-library';
+import {waitFor, within} from '@storybook/testing-library';
 import {withRouter} from 'storybook-addon-react-router-v6';
 
-import {withSubmissionPollInfo} from 'story-utils/decorators';
+import {AnalyticsToolsDecorator, withForm, withSubmissionPollInfo} from 'story-utils/decorators';
 
 import {ConfirmationViewDisplay} from './ConfirmationView';
 
 export default {
   title: 'Private API / Post completion views / Confirmation view',
   component: ConfirmationViewDisplay,
-  decorators: [withSubmissionPollInfo, withRouter],
+  decorators: [withSubmissionPollInfo, withRouter, AnalyticsToolsDecorator, withForm],
```

Comment:
Can you try to keep the order of decorators similar to what is actually rendered in the app? I think the router is typically the outermost decorator.

Storybook decorators are applied from [left-to-right](https://storybook.js.org/docs/writing-stories/decorators#decorator-inheritance), so leftmost is the inner and rightmost the outer wrapper.